users/danielo515/process_records.h

```diff
@@ -0,0 +1,105 @@
+#pragma once
+#include "quantum.h"
+
+enum custom_keycodes
+{
+  PLACEHOLDER = SAFE_RANGE, // can always be here
+    EPRM,
```

Comment:
This doesn't need a placeholder here.

```suggestion
    EPRM = SAFE_RANGE, 
```